src/paperstatus.php

```diff
@@ -761,7 +761,7 @@ static function check_abstract(PaperStatus $ps, $pj) {
         if ($v === ""
             && (isset($pj->abstract) || !$ps->prow || (string) $ps->prow->abstract === "")) {
             if (!$ps->conf->opt("noAbstract"))
-                $ps->error_at("abstract", $ps->_("Entry required."));
+                $ps->error_at("abstract", $ps->_("Abstract: Entry required."));
```

Comment:
Thanks for pointing out this problem. But the goal is for these error messages to *not* include field names, so that they make sense immediately below the relevant fields. The `PaperStatus::landmarked_messages()` method should prefix the message with the field name. But it didn’t know about the name of the abstract field. It does now, so I think this commit is redundant.

Reply:
Ok, this makes sense. Sorry about that! I see your fix from earlier today, and that's definitely a better approach.